### DISCUSSION TITLE
Looking for x64 dll for ClearScript

### DISCUSSION THREAD

**Discussion by susmitakanchinadham** [ClearFoundry/ClearScript#378] [Q&A]
Hi Team,

I'm looking for ClearScript x64 dll. I know it is available for Clearscriptv8.
Tried installing via nuget, but unable to get it due to VS2015 limitation.
"The 'Microsoft.ClearScript.V8.Native.win-x64 7.2.5' package requires NuGet client version '5.3.0' or above, but the current NuGet version is '3.4.4'. To upgrade NuGet, please go to http://docs.nuget.org/consume/installing-nuget"
Also tried building the source code, but facing issues loading the projects in VS2015.
Can you suggest any workaround to get x64 Clearscript dll. 

Thanks,
Susmita

**Comment by ClearScriptLib** [comment]
Hi Susmita,

You can download the package from NuGet ([link](https://www.nuget.org/api/v2/package/Microsoft.ClearScript.V8.Native.win-x64/7.2.5)), change its file extension from ".nupkg" to ".zip", and open the archive. You'll find the DLL inside, in the folder runtimes\win-x64\native.

Good luck!
